### ISSUE TITLE
Improve the implementation

### ISSUE THREAD

**Issue by Wu-Chenyang** [JuliaPOMDP/ParticleFilters.jl#48]
I suggest using the follwing codes to replace those in `policies.jl`. This will significantly improve the performance of `AlphaVectorPolicy`.
```julia
function unnormalized_util(p::AlphaVectorPolicy, b::AbstractParticleBelief)
    util = zeros(length(alphavectors(p)))
    for (i, s) in enumerate(particles(b))
        util .+= weight(b, i) .* getindex.(p.alphas, stateindex(p.pomdp, s))
    end
    return util
end
```

**Comment by zsunberg** [comment]
Hi Wu-Chenyang, Thanks for the suggestion! Can you submit a Pull Request?

**Comment by Wu-Chenyang** [comment]
Yes. I will submit a Pull Request tomorrow.

**Comment by lassepe** [comment]
Resolved by #49
